shortcuts.sh

```diff
@@ -132,32 +129,26 @@ function swgjs_create_amp_release_pr() {
 
   __swgjs_get_github_username
 
-  SWG_VERSION=$1
-  if [[ -z "$SWG_VERSION" ]]; then
-    echo "What SwG Version are you releasing? (ex: 0.1.22.333)"
-    read SWG_VERSION
-  fi
-
   swgjs_install
   __swgjs_install_amp
 
   # Build Swgjs for AMP.
   cd $SWGJS_PATH
   git fetch team
-  git checkout team/main
+  git checkout team/amp
   npx gulp build
-  npx gulp export-to-amp --swgVersion=$SWG_VERSION
+  npx gulp export-to-amp --swgVersion=AMP
```

Comment:
Is it ok for this to always be the same version going forward?

Reply:
Yes, I think that's fine. AMP has its own build/versioning system, and this script just exports a dependency for that

Reply:
Got it. Would it make sense for us to still maintain a version just so we can tie a specific export to a swg version? I imagine we should still create git tags for the amp versions, they should just be treated differently than the main line

Reply:
Good idea, I added the Git commit hash as a version